pkg/base1/cockpit.js

```diff
@@ -2248,6 +2248,22 @@ function full_scope(cockpit, $, po) {
         return fmt.replace(fmt_re, function(m, x, y) { return args[x || y] || ""; });
     };
 
+    cockpit.format_number = function format_number(number) {
+        /* non-zero values should never appear zero */
+        if (number > 0 && number < 0.1)
+            number = 0.1;
+        else if (number < 0 && number > -0.1)
+            number = -0.1;
+
+        /* TODO: Make the decimal separator translatable */
+
+        /* only show as integer if we have a natural number */
+        if (number % 1 === 0)
```

Comment:
I suspect isInteger() is old too, and I sure hope it works :) But to spend time actually discussing a change so minuscule...I'm confident that we both have more important things to do.